tests/testthat/test-shinytest2_filteredData_DF.R

```diff
@@ -0,0 +1,173 @@
+  testthat::expect_true(is_visible(app_driver, "#filter_panel"))
+  testthat::expect_true(is_visible(app_driver, "#filter_panel-overview-main_filter_accordion"))
+  testthat::expect_equal(app_driver$get_text("#filter_panel-overview-table > table > tbody > tr:nth-child(1) > td:nth-child(1)"),
+                         "iris")
+  testthat::expect_equal(app_driver$get_text("#filter_panel-overview-table > table > tbody > tr:nth-child(2) > td:nth-child(1)"),
+                         "mtcars")
+
+  testthat::expect_true(is_visible(app_driver, "#filter_panel-active"))
+  testthat::expect_true(is_visible(app_driver, "#filter_panel-active-iris-dataset_filter_accordion"))
+  testthat::expect_true(is_visible(app_driver, "#filter_panel-active-mtcars-dataset_filter_accordion"))
+
+})
+
+testthat::test_that("Toggle visibility of Active Filter Summary", {
+  app_driver <- app()
+
+  selector_collapsable <- get_class(".accordion-item:nth-of-type(1) > div.accordion-collapse")
+  out <- app_driver$get_js(selector_collapsable)
+  testthat::expect_length(out, 4)
+  testthat::expect_true("show" %in% unlist(out[[1]]))
+
+  selector <- ".filter-panel > .teal-slice:nth-of-type(1) .accordion-button"
+  app_driver$click(selector = selector)
+  app_driver$wait_for_idle(duration = default_idle_duration,
+                           timeout = default_idle_timeout)
+
+  out <- app_driver$get_js(selector_collapsable)
+  testthat::expect_length(out, 4L)
+  testthat::expect_true(!"show" %in% unlist(out[[1]]))
+  testthat::expect_equal(app_driver$get_text(
+    paste0("#filter_panel-overview-main_filter_accordion > div >",
+           " div.accordion-header > button > div.accordion-title")),
+    "Active Filter Summary")
+  app_driver$stop()
+})
+
+testthat::test_that("Toggle visibility of Filter Data", {
+  app_driver <- app()
+
+  selector_collapsable <- get_class(".accordion-item:nth-of-type(1) > div.accordion-collapse")
+  out <- app_driver$get_js(selector_collapsable)
+  testthat::expect_length(out, 4)
+  testthat::expect_true("show" %in% unlist(out[[2]]))
+
+  selector <- ".filter-panel > #filter_panel-active.teal-slice > div > div > div > button"
+  app_driver$click(selector = selector)
+  app_driver$wait_for_idle(duration = default_idle_duration,
+                           timeout = default_idle_timeout)
+
+  out <- app_driver$get_js(selector_collapsable)
+  testthat::expect_length(out, 4L)
+  testthat::expect_true(!"show" %in% unlist(out[[2]]))
+  testthat::expect_equal(
+    app_driver$get_text(paste0("#filter_panel-active-main_filter_accordion > div > ",
+                               "div.accordion-header > button > div.accordion-title")),
+    "Filter Data")
+
+  app_driver$stop()
+})
+
+testthat::test_that("Toggle visibility of filters for a dataset", {
+  app_driver <- app()
+
+  selector_collapsable <- get_class(".accordion-item:nth-of-type(1) > div.accordion-collapse")
+  out <- app_driver$get_js(selector_collapsable)
+  testthat::expect_length(out, 4)
```

Comment:
Either use `4L` on all or just `4`.

```suggestion
  testthat::expect_length(out, 4L)
```